superset/charts/commands/export.py

```diff
@@ -34,7 +34,7 @@
 
 
 # keys present in the standard export that are not needed
-REMOVE_KEYS = ["datasource_type", "datasource_name", "query_context", "url_params"]
+REMOVE_KEYS = ["datasource_type", "datasource_name", "url_params"]
```

Comment:
Why did you add the `query_context` to the payload of exported charts?

Reply:
@michael-s-molina this logic was extracted from https://github.com/apache/superset/pull/23012.